### PR TITLE
Add traffic for the aggregate and some fixes

### PR DESCRIPTION
**Pull request by DanG100**
Traffic part copied from https://github.com/openconfig/featureprofiles/blob/main/feature/interface/aggregate/otg_tests/balancing_test/balancing_test.go

Other changes:
* IP2ME support for aggregate interface
* Fixes Lag saiserver API

### CODE REVIEW

**Comment by DanG100** [comment]
need to investigate more, this test passing consistently locally but failing remote

File: dataplane/saiserver/ports.go [hunk 1]
```diff
@@ -426,7 +454,8 @@ func (l *lag) CreateLagMember(ctx context.Context, req *saipb.CreateLagMemberReq
 		Update: &fwdpb.PortUpdateDesc{
 			Port: &fwdpb.PortUpdateDesc_AggregateAdd{
 				AggregateAdd: &fwdpb.AggregatePortAddMemberUpdateDesc{
-					PortId: &fwdpb.PortId{ObjectId: &fwdpb.ObjectId{Id: fmt.Sprint(req.GetPortId())}},
+					InstanceCount: 1,
```
Comment (wenovus) [inline dataplane/saiserver/ports.go:457]: What is InstanceCount?

File: dataplane/dplanerc/interface.go [hunk 2]
```diff
@@ -411,7 +424,20 @@ func (ni *Reconciler) removeLAGMember(ctx context.Context, intf ocInterface, mem
...
+	aggID := ni.getOrCreateInterface(intf.name).GetOrCreateEthernet().GetAggregateId()
 	ni.getOrCreateInterface(intf.name).GetOrCreateEthernet().AggregateId = nil
+	idx := -1
+	for i, member := range ni.getOrCreateInterface(aggID).GetOrCreateAggregation().Member {
+		if member == intf.name {
+			idx = i
+			break
+		}
+	}
+	if idx != -1 {
```
Comment (wenovus) [inline dataplane/dplanerc/interface.go:436]: if isAggregateMember := idx != -1; isAggregateMember {

File: dataplane/dplanerc/interface.go [hunk 3]
```diff
@@ -710,13 +745,31 @@ func (ni *Reconciler) handleAddrUpdate(ctx context.Context, au *netlink.AddrUpda
 			gnmiclient.BatchUpdate(sb, ocpath.Root().Interface(intf.name).Subinterface(intf.subintf).Ipv6().Address(ip).Ip().State(), au.LinkAddress.IP.String())
 			gnmiclient.BatchUpdate(sb, ocpath.Root().Interface(intf.name).Subinterface(intf.subintf).Ipv6().Address(ip).PrefixLength().State(), uint8(pl))
 		}
-		_, err := ni.fwdClient.TableEntryAdd(ctx, fwdconfig.TableEntryAddRequest(ni.contextID, cpusink.IP2MeTable).
-			AppendEntry(entry, fwdconfig.Action(fwdconfig.TransmitAction(fmt.Sprint(data.hostifID)))).Build())
-		if err != nil {
-			log.Warningf("failed to add route: %v", err)
-			return
+		// For an aggregate interface, add an IP2ME route for all of it's members.
```
Comment (wenovus) [inline dataplane/dplanerc/interface.go:748]: it's -> its

File: dataplane/dplanerc/interface.go [hunk 4]
```diff
@@ -737,29 +790,50 @@ func (ni *Reconciler) handleAddrUpdate(ctx context.Context, au *netlink.AddrUpda
...
 		}
-		_, err := ni.fwdClient.TableEntryRemove(ctx, &fwdpb.TableEntryRemoveRequest{
-			ContextId: &fwdpb.ContextId{Id: ni.contextID},
-			TableId:   &fwdpb.TableId{ObjectId: &fwdpb.ObjectId{Id: cpusink.IP2MeTable}},
-			EntryDesc: entry.Build(),
-		})
-		if err != nil {
-			log.Warningf("failed to remove route: %v", err)
-			return
+		// For an aggregate interface, add an IP2ME route for all of it's members.
```
Comment (wenovus) [inline dataplane/dplanerc/interface.go:793]: Is this part of the logic duplicate with the above?

File: dataplane/saiserver/ports.go (hunk 1 above)
Comment (DanG100) [reply dataplane/saiserver/ports.go:457]: it's the number of copies of the port to add to the group (before this was adding 0 copies)

File: dataplane/dplanerc/interface.go (hunk 2 above)
Comment (DanG100) [reply dataplane/dplanerc/interface.go]: done

File: dataplane/dplanerc/interface.go (hunk 4 above)
Comment (DanG100) [reply dataplane/dplanerc/interface.go:793]: somewhat, do it's calling different func with a different request type. i didn't see a good to dedupe it

File: dataplane/saiserver/ports.go (hunk 1 above)
Comment (wenovus) [reply dataplane/saiserver/ports.go:457]: Interesting, what does adding multiple copies of the same port to the group mean?

**Review by wenovus — APPROVED** [review]

File: dataplane/saiserver/ports.go (hunk 1 above)
Comment (DanG100) [reply dataplane/saiserver/ports.go:457]: you could unevenly weight the ports in the group i guess? 
send 75% of traffic to port 1 and 25% to port 2. Not sure that's practical use case for this

Comment (wenovus) [reply dataplane/saiserver/ports.go:457]: Alright, if it passes the test and satisfy the use case you're going for then LGTM.

**Pull request merged**